packages/salesforcedx-apex-debugger/src/adapter/apexDebug.ts

```diff
@@ -1113,6 +1135,7 @@ export class ApexDebug extends LoggingDebugSession {
       );
     }
 
+    this.sendEvent(new Event(RESET_HEARTBEAT_EVENT));
```

Comment:
This is in the method variablesRequest(). We use this to tell the extension to reset the heartbeat timer.